server/glsp-api/src/main/java/com/eclipsesource/glsp/api/utils/ServerStatusUtil.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright (c) 2017-2018 TypeFox and others.
+ * Copyright (c) 2019 EclipseSource and others.
+ *  
+ *   This program and the accompanying materials are made available under the
+ *   terms of the Eclipse Public License v. 2.0 which is available at
+ *   http://www.eclipse.org/legal/epl-2.0.
+ *  
+ *   This Source Code may also be made available under the following Secondary
+ *   Licenses when the conditions for such availability set forth in the Eclipse
+ *   Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ *   with the GNU Classpath Exception which is available at
+ *   https://www.gnu.org/software/classpath/license.html.
+ *  
+ *   SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ******************************************************************************/
+package com.eclipsesource.glsp.api.utils;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+import com.eclipsesource.glsp.api.action.kind.ServerStatusAction;
+import com.eclipsesource.glsp.api.types.ServerStatus;
+import com.eclipsesource.glsp.api.types.ServerStatus.Severity;
+
+public class ServerStatusUtil {
+
+	public static ServerStatusAction info(String message) {
+		return new ServerStatusAction(new ServerStatus(Severity.INFO, message));
+	}
+
+	public static ServerStatusAction warn(String message) {
+		return new ServerStatusAction(new ServerStatus(Severity.WARNING, message));
+	}
+
+	public static ServerStatusAction error(String message, String details) {
+		return new ServerStatusAction(new ServerStatus(Severity.ERROR, message, details));
+	}
+
+	public static ServerStatusAction error(Exception e) {
+		return error(e.getMessage(), printStacktrace(e.getCause()));
```

Comment:
If there is no message then the client will show `undefined`, cf. screenshot below. Maybe we could show the class name if there is no message so that we know at least what type of exception it is, e.g., this happened with a NPE:

![Selection_087](https://user-images.githubusercontent.com/19170971/62539566-c3e7f380-b855-11e9-8927-37c903580f21.png)